flashlight/fl/tensor/backend/jit/JitTensorBase.h

```diff
@@ -81,6 +79,10 @@ class JitTensorBase : public TensorAdapterBase {
 
   /**
    * Return the node this JIT tensor represents.
+   * NOTE
+   * 1. `const` w.r.t. the underlying Tensor this represents.
+   * 2. return a node that represents the underlying Tensor, i.e., if there is
+   *    indexing, take care of it.
```

Comment:
> take care of it

Can you elaborate a little -- does that mean realize indexing ops into the graph?

Reply:
Duplicated comment:)